static/src/stylesheets/module/onward/_breaking-news.scss

```diff
@@ -72,7 +68,7 @@ $breaking-news-icon-width: 36px;
 
 .breaking-news__item-content {
     @include box-sizing(border-box);
-    padding: $gs-baseline 0 $gs-baseline ($gs-gutter + $breaking-news-icon-width);
+    padding: $gs-baseline * .5 0 $gs-baseline ($gs-gutter + $breaking-news-icon-width);
```

Comment:
Might be worth bracketing this up for legibility. Seeing as you do it later on the line.